packages/@aws-cdk/aws-servicecatalog/lib/private/validation.ts

```diff
@@ -0,0 +1,15 @@
+/**
+ * Abstract class to validate inputs match requirements.
+ */
+export class InputValidator {
+/**
+* Validates length
+*/
+  public static validateLength(inputName: string, inputString: string | undefined, minLength: number, maxLength: number): void {
```

Comment:
Let's just use functions for these, instead of a class with a single static method.

Reply:
This class will have ~10 more functions added in next pull request, trying to keep importing cleaner for future